docs/rest-api/getting-started/graph-oauth.md

```diff
@@ -126,7 +126,7 @@ GET https://login.microsoftonline.com/common/oauth2/v2.0/authorize?client_id={cl
 | Parameter name   | Value    | Description                                                                                           |
 | ---------------- | -------- | ----------------------------------------------------------------------------------------------------- |
 | *client_id*      | string   | The client ID created for your app.                                                                   |
-| *scope*          | string   | A space-separated list of scopes that your app requires.                                              |
+| *scope*          | string   | A space-separated list of scopes that your app requires. If you want use "offline_access", you should pass it at first.                                              |
```

Comment:
This looks like a bug.  Have you reached out to the AAD team about this?  https://stackoverflow.com/questions/tagged/azure-active-directory

Reply:
The rest of the proposed changes look good.